creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-useless-return */
+import changeBg from '../../features/changeBg/changeBg.js';
```

Comment:
Please keep import statement inside if changeBG condition below so that it is not imported for all interactive marquee variations. this is only if we keep this marquee as a variation of interactive variation

Reply:
done